packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -98,28 +91,28 @@ export const AnonymityLevelSlider = ({ className }: AnonymityLevelSliderProps) =
             );`,
     };
 
+    const isErrorDisplayed = anonymityStatus === AnonymityStatus.Bad;
+
     return (
         <Container className={className}>
             <Slider
                 value={sliderPosition}
                 onChange={handleSliderChange}
                 trackStyle={trackStyle}
                 step="any"
+                labels={[1, 3, 10, 30, 100]}
+                onLabelClick={number => setAnonymity(getPosition(number))}
             />
-            <LabelsWrapper>
-                <Label>1</Label>
-                <Label>3</Label>
-                <Label $offset={7}>10</Label>
-                <Label>30</Label>
-                <Label>100</Label>
-            </LabelsWrapper>
-            {anonymityStatus === AnonymityStatus.Bad && (
-                <WarningWrapper>
-                    <Warning critical withIcon>
-                        <Translation id="TR_ANONYMITY_LEVEL_BAD_WARNING" />
-                    </Warning>
-                </WarningWrapper>
-            )}
+
+            <AnimatePresence initial={!isErrorDisplayed}>
```

Comment:
Currently, only `AnonymityStatus.Bad` is used, but the other statuses should probably be displayed as well.

Reply:
will be solved in another PR